apps/web/app/(app)/onboarding/OnboardingBulkUnsubscriber.tsx

```
@@ -0,0 +1,144 @@
+"use client";
+
+import useSWR from "swr";
+import Link from "next/link";
+import { usePostHog, PostHog } from "posthog-js/react";
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)